test/alternator/test_metrics.py

```diff
@@ -130,6 +131,31 @@ def test_batch_write_item(test_table_s, metrics):
         test_table_s.meta.client.batch_write_item(RequestItems = {
             test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}]})
 
+def test_batch_write_item_histogram(test_table_s, metrics):
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '2.000000'}], [1, {'op': 'BatchWriteItem', 'le': '3.000000'}]]):
+        test_table_s.meta.client.batch_write_item(RequestItems = {
+            test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}, {'PutRequest': {'Item': {'p': random_string(), 'a': 'hi1'}}}, {'PutRequest': {'Item': {'p': random_string(), 'a': 'hi1'}}}]})
+    items = [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}} for i in range(100)]
+    items2 = [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}} for i in range(100)]
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '86.000000'}], [2, {'op': 'BatchWriteItem', 'le': '103.000000'}]]):
+        test_table_s.meta.client.batch_write_item(RequestItems = {
+            test_table_s.name: items})
+        test_table_s.meta.client.batch_write_item(RequestItems = {
+            test_table_s.name: items2})
+
+def test_batch_get_item_histogram(test_table_s, metrics):
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '2.000000'}], [1, {'op': 'BatchGetItem', 'le': '3.000000'}]]):
+        test_table_s.meta.client.batch_get_item(RequestItems = {
+            test_table_s.name: {'Keys': [{'p': random_string()}, {'p': random_string()}, {'p': random_string()}], 'ConsistentRead': True}})
+    keys = [{'p': random_string()} for i in range(100) ]
+    keys2 = [{'p': random_string()} for i in range(100) ]
```

Comment:
Here, the number 100 is fine (this is the maximum allowed in BatchGetItem in DynamoDB), but how much time does this test take? You have 200 cache misses here :-( Again I don't know if we need so many for the test.

Reply:
I didn't time it, but it didn't seem slow; as you know from my previous metrics tests, I prefer testing actual values with the potential overhead that tests would break if implementation change

Reply:
Yes, but I 100 isn't a typical "actual value" - it's actually way beyond the maximum in DynamoDB (which is 25) and only recently we agreed that 100 will be the maximum in Scylla (if you tried 200, this test would start to error out).

Anyway, I just measured `test/alternator/run  test_metrics.py::test_batch_write_item_histogram` and it takes 0.2 seconds on my machine. Not speediest test - it's 20% of the entire file despite being just one of 20 tests - but it's also not slow enough to worry about. So ok.